Microsoft.PowerShell_profile.ps1

```diff
@@ -192,8 +192,12 @@ function unzip ($file) {
     $fullFile = Get-ChildItem -Path $pwd -Filter .\cove.zip | ForEach-Object { $_.FullName }
     Expand-Archive -Path $fullFile -DestinationPath $pwd
 }
-function ix ($file) {
-    curl.exe -F "f:1=@$file" ix.io
+function hb ($file) {
+    $response = curl.exe -X POST -T "$file" "https://hastebin.skyra.pw/documents"
+    $object = $response | ConvertFrom-Json
+    $key = $object.key
+    $url = "https://hastebin.skyra.pw/$key" 
+    Write-Output $url
```

Comment:
I do not think we should be adding this. I'm aware that it increases usability but it does make calls to a third-party website which is not intended.  
If it tomorrow goes down, then you cannot use it until either it is up or you find another alternative.   

This would also mean that you will need an active internet connection to use this command.

I would rather suggest modifying this such the work gets done within powershell without making calls.

Reply:
I see, Thx for the feedback